inc/Calculator.h

```diff
@@ -11,6 +11,55 @@
 #define CALCULATOR_H
 
 #include <iostream>
+#include <string>
+#include <sstream>
+#include <vector>
 // Add your function declarations here.
 
-#endif  // CALCULATOR_H
\ No newline at end of file
+class Calculator {
+    public:
+
+        //when adding Operations make sure to add them to validOperations_ as well
+        enum class Operation: char
+        {
+            ADD      = '+',
+            DIVIDE   = '/',
+            MULTIPLY = '*',
+            SUBTRACT = '-'
+        };
+
+        Calculator();
```

Comment:
Look at the other PR for comments on constructor and function, regardless if they are public or private